projects/angular/src/data/tree-view/tree-node.ts

```diff
@@ -190,6 +201,20 @@ export class ClrTreeNode<T> implements OnInit, OnDestroy {
     );
   }
 
+  ngAfterContentInit() {
+    if (this.forTypeAheadProvider) {
+      this._model.textContent = this.forTypeAheadProvider.textContent;
+
+      this.subscriptions.push(
+        this.typeAheadKeyEvent.pipe(debounceTime(TREE_TYPE_AHEAD_TIMEOUT)).subscribe((bufferedKeys: string) => {
+          this.focusManager.focusNodeStartsWith(bufferedKeys, this._model);
+          // reset once bufferedKeys are used
+          this.typeAheadKeyBuffer = '';
+        })
+      );
+    }
+  }
+
   ngOnDestroy() {
     this._model.destroy();
     this.subscriptions.forEach(sub => sub.unsubscribe());
```

Comment:
It might be worth taking a look at [Artur's subscribe/unsubscribe work for consistency](https://github.com/vmware-clarity/ng-clarity/pull/9).